reana_client/api/client.py

```diff
@@ -928,10 +976,10 @@ def add_secrets(secrets, overwrite, access_token):
     """Add new secrets.
 
     :param secrets: dictionary containing all the secrets to be sent.
-      The dictionary with secret names for keys and for each key there is
-       a dictionary with two fields:
-      - 'value':  a base64 encoded file or literal string
-      - 'type': 'file' or 'env'
+        The dictionary with secret names for keys and for each key there is
+        a dictionary with two fields:
+        - 'value':  a base64 encoded file or literal string
+        - 'type': 'file' or 'env'
```

Comment:
This is not displayed as a list: https://reana-client--669.org.readthedocs.build/en/669/#reana_client.api.client.add_secrets

What about either making the list work if possible, or rephrasing this to avoid the bullet points?